packages/hint-compat-api/README.md

```diff
@@ -12,6 +12,8 @@ are compatible with the target browsers that you want to support.
 
 * [compat-api-css][compat-api-css]
 * [compat-api-css-next][compat-api-css-next]
+* [compat-api-html][compat-api-html]
+* [compat-api-html-next][compat-api-html-next]
```

Comment:
Also, we should probably switch the displayed link text from `compat-api-css` to `compat-api/css` (and similar).

Reply:
I'm going to update these ids but leave the rest untouched because you are dealing with them in your PR and don't want to cause any unnecessary conflicts 😊